solidity/core/contracts/Common.sol

```diff
@@ -68,10 +68,7 @@ abstract contract Common is ICommon, OwnableUpgradeable {
 
     // ============ Initializer ============
 
-    function __Common_initialize(address _validatorManager)
-        internal
-        initializer
-    {
+    function __Common_initialize(address _validatorManager) internal {
```

Comment:
Why remove `initializer` here?

Reply:
the new version of `Initializable` has the only top-level requirement

Reply:
https://github.com/abacus-network/abacus-monorepo/pull/447